guide/14-deep-learning/track_objects_using_SiamMask.ipynb

```diff
@@ -100,7 +100,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Can we use SiamMask model object instead of SiamMask class object?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Done</p>